src/neural/network_opencl.cc

```diff
@@ -134,111 +134,102 @@ class OpenCLNetwork : public Network {
     params_.tune_exhaustive =
         options.GetOrDefault<bool>("tune_exhaustive", false);
 
-    const int inputChannels = kInputPlanes;
     const int channels = weights.input.biases.size();
-    const size_t residual_blocks = weights.residual.size();
 
-    /*
-     static constexpr int NUM_VALUE_INPUT_PLANES = 32;
-     static constexpr int NUM_POLICY_INPUT_PLANES = 32;
-     static constexpr int NUM_OUTPUT_POLICY = 1858;
-     static constexpr int NUM_VALUE_CHANNELS = 128;
-     */
+    ////////////////////////////////////////////////////////////////////////////
+    // Input convolution
 
-    int NUM_VALUE_INPUT_PLANES = weights.value.bn_means.size();
```

Comment:
The naming scheme of these variables was not correct but you could have let them live. 
Materializing the quantities was important for someone who as reading/writing the code.
Now, with these 4 temporary variables gone, the code is not faster bu much less unreadable.
It will be even harder to add batching or alike, without undoing this change, and make these useful variables appear again.

```
int num_value_input_planes = weights.value.bn_means.size();
int num_policy_input_planes= weights.policy.bn_means.size();
int num_output_policy = weights.ip_pol_b.size();
int num_value_channels = weights.ip1_val_b.size();
```

Reply:
See my other recent comments; I very much disagree that it's more readable, I think it's far more confusing to have separate variables that add no calculation. And referring to block weight sizes would have made far more sense to me when I was reading lczero code the first time than to random "num input planes" or "num policy channels" or whatever, which make no reference to which weights they are about.

Reply:
"far more confusing". 

The code is written for the machine, but also for humans to read it back later.

This is not because `int num_output_policy = weights.ip_pol_b.size();` add *no calculation* that you can remove it.

No only the remaining portion of code is more readable but it is shorter.  Because everywhere you will need `num_output_policy, you won't have to write `weights.ip_pol_b.size()`.

It's very difficult to understand you disagreeing here, as it is pretty much common sense.